pkg/exporter/sumologicexporter/README.md

```diff
@@ -17,7 +17,10 @@ exporters:
     # if sumologicextension is not being used, the endpoint is required
     endpoint: <HTTP_Source_URL>
     # Compression encoding format, empty string means no compression, default = gzip
+    # NOTICE: compress_encoding is deprecated and will be removed in an upcoming release
     compress_encoding: {gzip, deflate, ""}
+    # Compression encoding format, empty string means no compression, default = gzip
+    compression: {gzip, zstd, deflate, ""}
```

Comment:
Do we need to deprecate it? Does it make sense to reuse it. It would just add zstd to `compress_encoding`

Reply:
Ok, I see it is easier that way, and actually we already support `compress` from confighttp helper